intranet/static/js/hoco_scores.js

```diff
@@ -1,13 +1,18 @@
 $(document).ready(function() {
-    $.get("https://homecoming.tjhsst.edu/api/", function(data) {
-        console.log(data.senior_total ? data.senior_total : 0);
-        $("#score-senior").text(data.senior_total ? data.senior_total : 0);
-        $("#score-sophomore").text(data.sophomore_total ? data.sophomore_total : 0);
-        $("#score-junior").text(data.junior_total ? data.junior_total : 0);
-        $("#score-freshman").text(data.freshman_total ? data.freshman_total : 0);
-        giveRibbons();
-        $("#hoco-scores").fadeIn();
-    });
+    function loadScores() {
+        $.get("https://homecoming.tjhsst.edu/api/", function(data) {
+            console.log(data.senior_total ? data.senior_total : 0);
+            $("#score-senior").text(data.senior_total ? data.senior_total : 0);
+            $("#score-sophomore").text(data.sophomore_total ? data.sophomore_total : 0);
+            $("#score-junior").text(data.junior_total ? data.junior_total : 0);
+            $("#score-freshman").text(data.freshman_total ? data.freshman_total : 0);
+            removeRibbons();
+            giveRibbons();
+        });
```

Comment:
In case the network is slow or unavailable, this AJAX query may fail. If it fails more than some number of times in a row, say 3, then I think it should display a message that it was unable to query hoco scores to avoid misinforming people.

I would implement this with a counter and `jQuery.ajax()`'s error attribute.

Reply:
Done in c718222 - it just hides hoco scores if the API is unreachable, and unhides them when it's reachable again.

Reply:
Please implement described counter and also squash commits.